src/app/main.cpp

```diff
@@ -113,6 +113,24 @@ typedef SInt32 SRefCon;
 #include "qgsopenclutils.h"
 #endif
 
+/**
+ * Print QGIS version
+ */
+void version( )
+{
+  const QString msg = QStringLiteral( "QGIS %1 '%2' (%3)\n" ).arg( VERSION ).arg( RELEASE_NAME ).arg( QGSVERSION );
+
+#ifdef Q_OS_WIN
+  MessageBox( nullptr,
+              msg,
+              "QGIS version",
+              MB_OK );
+#else
+  std::cerr << msg.toStdString();
```

Comment:
I see that you just copied from the help output, but do you know why `cerr` and not `cout` ?

Reply:
Good point @elpaso 

I would prefer to have both `--version` and `--help` going to `cout` and both returning `EXIT_SUCCESS`. I don't know why calling `qgis --help` is returning 2.

If the program is called with incorrect syntax, it does make sense to write usage on `cerr`, but only if that happens.

I'll make the changes. Thank you for your comment.